L/libjulia/common.jl

```diff
@@ -286,16 +285,17 @@ function build_julia(ARGS, version::VersionNumber)
         push!(dependencies, Dependency("LibOSXUnwind_jll", compat="0.0.5"))
     elseif version < v"1.7"
         push!(dependencies, Dependency("LibOSXUnwind_jll", compat="0.0.6"))
+        push!(dependencies, Dependency("LibUV_jll"))
     end
 
     if version < v"1.6"
-        push!(dependencies, BuildDependency("SuiteSparse_jll", compat="5.4.0"))
+        push!(dependencies, BuildDependency(PackageSpec(name="SuiteSparse_jll", version="5.4.0")))
```

Comment:
Could you briefl say what this change is about?

Reply:
`BuildDependency` has no compat keyword, so you need to set it using `PackageSpec`. The original PR didn't touch the `common.jl` file, but it seems the build process uses `common.jl` from master, so I had to fix this here.